### ISSUE TITLE
compile error: unknown symbol "./data/insta/html"

### ISSUE THREAD

**Issue by xhh** [bakpakin/mendoza#5]
When running `mdz` after installing, I got this error:

    compile error: unknown symbol "./data/insta/html" on line 124

The related code is:

```janet
(import ./data/insta) # <=== Changing to `(import ./data/insta :as ./data/insta)` fixes the error
(defn instagram
  "Add an instagram link in the page"
  [id]
  {:tag "div"
   "class" "mendoza-video"
   :content {:no-escape (string/replace "!!!!!" (string id) ./data/insta/html)}}) # <=== Line 124 here
```

I'm using 1.7.0-f7ee8bd under Windows 10. Does default alias of relative imports only work with Linux or the latest Janet version?

**Comment by bakpakin** [comment]
Mendoza just needs to be updated, as import rules were changed in 1.7.0.

**Comment by bakpakin** [comment]
Fixed in 5ed7248c165f451af237f0b32a22443640907b5b
